src/Cli/dotnet/Commands/Help/HelpCommandParser.cs

```diff
@@ -2,9 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.CommandLine;
-using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Tools.Help;
```

Comment:
Isn't this overriden in `src/Cli/dotnet/Commands/Help/HelpCommand.cs`?

Reply:
This isn't related to that. This is likely related to the `LocalizableStrings.resx`, which will be handled in a separate PR. https://github.com/dotnet/sdk/blob/bc28cd5f0fe0e35e15b8280361a09ad0c6935a58/src/Cli/dotnet/dotnet.csproj#L50